### ISSUE TITLE
Lokuttaravada Vinaya texts are not showing up

### ISSUE THREAD

**Issue by sujato** [suttacentral/legacy-suttacentral#147]
There's problems with the display of the various texts in the Lokuttaravada Vinaya. Many of them don't show up, this is because of peculiarities in the texts. For example, the sekhiyas only appear in an uddana, so just as a list, so they can't be presented as separate files. Each Sekhiya rule should link to the whole text. There are several issues like this that need sorting, when someone gets some time let me know and we can get to it.

**Comment by ghost** [comment]
This is the same as issue #139
I was working on it but became distracted with lots of other things so I put it on the issue list.

**Comment by ghost** [comment]
Can you tell me where the Nidana belongs in this? It now shows up in two placed:
Lokuttaravāda Nidāna and Lokuttaravāda Miscellaneous
All the rest is done.

**Comment by sujato** [comment]
It's the introduction to the garudhammas, so it occurs immediately before gd1.

**Comment by ghost** [comment]
What I mean is, that in the menu it appears twice so I need to know which one is the correct place or do you want to keep them both:
https://suttacentral.net/skt-lo-bi-vb-nidana
or
https://suttacentral.net/skt-lo-vi-misc

The Garudhammas are in the same document underneath the origin story. I can split these up if you like or be kept in the same document.

**Comment by sujato** [comment]
Okay, that's a mistake, get rid of the misc category, not sure what's going on there.

Also, notice that both the words NIDANA and MISC are still showing up with the ALLCAPS bug. this is also still apparent on the search results for THA Ap and THI Ap, as well as the search results for NIDANA

**Comment by ghost** [comment]
Please have a look - now changed, also the allcaps bug in there but I cannot do anything about the allcaps bug in Elasticsearch. 
Have a look if this is OK like this ... also at GD 8 because the story continues underneath. I can make this into a separate chapter or add all chapters back together in one document

**Comment by sujato** [comment]
Okay, something's gone badly wrong here. We've lost a huge chunk of text. My apologies for not keeping a closer eye on what's happening, so I am not sure if this was the case earlier, or what is going on.

One of the key features of this text is that it gives a lengthy analysis of each of the garudhammas, something that is lacking in most Vinayas. This is why each of the garudhammas has it's own page, it is for the vibhanga. However this has vanished and we now just have the plain rules ,and cut off before the vibhanga starts. In fact all the text we currently have here: https://suttacentral.net/skt-lo-bi-nidana should be included in one file, the Nidana, which includes the plain list of garudhammas. Then we have eight separate files, one for each of the vibhangas for the garudhammas, and named gd1, gd2, and so on, not "Nidana". A nidana is an introduction, you can never have a numbered Nidana. I will email you the correct text, these are the files I originally prepared for SC.

**Comment by ghost** [comment]
The Nidana was indeed one text but I split it up. I was not sure about that so I asked you and I can put it back into one text. No text was actually deleted there. The rest is as it was before you started this post.

**Comment by sujato** [comment]
Okay, as long as the full text is there as per my email.

**Comment by ghost** [comment]
OK. Now it is back the way it was on the site with Nidana first so have a look. However, when I look at your sort-order file you just sent me, this is not exactly what you want.
So if I read that file correctly, what you want is the following and not Nidana on its own above Bhikkhuni Vibhanga. At least this would make sense to me. So please confirm if that is correct.

```
Bhikkhuni Vibhanga
- Nidana
- Garudhammas
- rest i.e. PJ, SS, etc.
```

**Comment by sujato** [comment]
So please confirm if that is correct.

Bhikkhuni Vibhanga
- Nidana
- Garudhammas
- rest i.e. PJ, SS, etc.

Exactly! On https://suttacentral.net/skt-lo-bi-nidana we should have a list of the garudhammas, as you had before, but linking to the full vibhanga texts. They then lead on to the parajika, etc.

**Comment by ghost** [comment]
The list of the Garudhammas is now at the bottom of the Vibhanga list so the texts were always there, just in the wrong order: https://suttacentral.net/skt-lo-bi-vb

I will change it.

**Comment by sujato** [comment]
Ahh, okay, that's where they got to! Sneaky little buggers

**Comment by ghost** [comment]
Voila!
Pls close issue when you're happy with this.

**Comment by sujato** [comment]
Ayya, I am so sorry, I just realized I made a mistake earlier. The Nidana & Garudhammas should remain as a separate item in the menu, preceding the Vibhanga, so we should have:
- **Nidana**
  - gd1
  - gd2 … and so on
- **Bhikkhuni Vibhanga**
  - Parajika …

With the actual menu items bolded, and other items available on the division view.

I realize that i confused the issue by speaking of the "vibhanga" of the garudhammas, this was just meant in a general sense. The section actually named "Bhikkhuni Vibhanga" should remain, as in every other Vinaya, the analysis of the patimokkha rules only. The garudhammas are separate, and in this text form part of the introduction to the whole Bhikkhuni Vinaya. 

meanwhile, i thought we had parallels for the garudhammas, but it seems not. On the 2-do list …
